src/main/java/com/kobylchak/bookstore/BookStoreApplication.java

```diff
@@ -1,11 +1,33 @@
 package com.kobylchak.bookstore;
 
+import com.kobylchak.bookstore.model.Book;
+import com.kobylchak.bookstore.service.BookService;
+import java.math.BigDecimal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class BookStoreApplication {
+    @Autowired
+    private BookService bookService;
```

Comment:
Lets use lombok for constructor creating
```suggestion
    private BookService bookService;
```